src/components/ImageView/Toolbar/ToolbarComponent.tsx

```diff
@@ -69,6 +73,14 @@ export class ToolbarComponent extends React.Component<ToolbarComponentProps> {
             </Menu>
         );
 
+        const coordinateSystemMenu = (
+            <Menu>
+                {Object.keys(SystemType).map(key => (
+                    <MenuItem text={key} onClick={() => this.handleCoordinateSystemClicked(key)}/>
```

Comment:
See [here](https://github.com/CARTAvis/carta-frontend/issues/538#issuecomment-534134257), we should put shorthand in, instead of the full names. 

Also, I think we should show the "full" descriptions in the tooltip, using a similar styling approach to the zoom buttons (_Current: 1.0x_)

Given these requirements, I suggest you hard-code the systems menu, or add an additional map object, instead of mapping straight from `SystemType` (although that's a nice elegant way to do so. It's very unlikely that we'll change the system types in the near future.

Reply:
Can we somehow combine the grid button with the reference frame button, similar to the region button?

Reply:
Can we somehow combine the grid button with the reference frame button, similar to the region button?

Reply:
Fixed

Reply:
Ok, let’s try two buttons.

Reply:
Ok, let’s try two buttons.

Reply:
Fixed